### DISCUSSION TITLE
Permissions error setting up local install

### DISCUSSION THREAD

**Discussion by jules-o** [broadinstitute/seqr#1946] [Q&A] [answered]
Hi,

I'm trying to set up a new local install with docker on Ubuntu 20.04, but the elasticsearch container exits straight away as "unhealthy". Elasticsearch container logs show permission denied error (copy attached).

If I change the permissions on the data directory and try again then setup appears to complete and I can create the superuser, but then I get a blank page when I go to localhost - log from seqr container also attached. 

I have LZMA command line tool 9.22 and LZMA SDK 9.22

What am I doing wrong?

[docker_elasticsearch_log.txt](https://github.com/broadinstitute/seqr/files/6805081/docker_elasticsearch_log.txt)
[docker_seqr_log.txt](https://github.com/broadinstitute/seqr/files/6805082/docker_seqr_log.txt)

**Comment by hanars** [comment]
So it sounds like the issue with elasticsearch was the directory permission, and you fixed that. For seqr, the logs actually show a successful 200 response for the status endpoint, which means all dependent services including elasticsearch and your database are running properly, which is good. I wonder if the issue is actually an error in the javascript, as I'm not seeing anything in the server side logs. Can you try and send me a screenshot/ copy-paste of the console output from the browser?

**Comment by jules-o** [comment]
What do you mean by the console output from the browser? Attached is the blank screen I get at localhost. Also tried going to kibana at http://localhost:5601/app/kibana#/home?_g=() and it says can't connect to server.

![seqr_home](https://user-images.githubusercontent.com/41348829/125494001-a7115e3c-3ef6-484d-b362-6430aafd287e.png)

**Comment by hanars** [comment, marked as answer]
Right click on the page, click "inspect element" and look at the chrome console

**Comment by jules-o** [comment]
Oh I see - sorry for being thick! Looks like Firefox was blocking the cookies, even after I told it not to. Just installed chrome and it's working fine. Thank you for helping!

**Reply by hanars** [reply]
glad that helped! seqr actually is only supported in chrome, so for future reference we recommend you use that browser

**Reply by jules-o** [reply]
OK will do. It seems to be an issue with the Content Security Policy in Firefox so I may be able to change the settings with that, but good to know it is working properly :)

**Comment by jules-o** [comment]
Hi Hana, sorry to bother you again but I must be making another mistake. I've loaded my data and all seems good, except that there is no mention of HGMD in the variant search filters, only Clinvar. My dataset does have HGMD annotations and I have previously used it with another seqr local install, where the option to filter by HGMD pathogenicity was there. What have I missed?

**Reply by hanars** [reply]
For our installation of seqr, we have made HGMD available only to our internal analysts, as our HGMD license does not permit us to share HGMD with our external collaborators. This means that even if HGMD is in the data, we only show it/ allow searching on it to a subset of our users. It appears we did not make this available by default to users with local installations, as we don't make our HGMD annotations public and I guess weren't considering the use where users had their own HGMD license.

We have planned work to make HGMD available on a per-project basis, so you could just enable it on all your projects, but that won't be done for a few weeks: https://github.com/broadinstitute/seqr/issues/1958

In the meantime, if you define the environment variable `ANALYST_USER_GROUP` in your docker compose, create a UserGroup with that same name, and add yourself to that user group you should see all the HGMD filters and annotations again

**Reply by jules-o** [reply]
Glad it wasn't just me doing something wrong! Thank you, all working again now. By the way, I'd just like to say how much I appreciate seqr. It's sooooooo much better than the (several) other tools I've tried, and I'm very grateful for the active support.
